api/v1alpha1/quaysecretenginestaticrole_types.go

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
so a static tole is a role with no expiration? Why can't you handle that with an special TTL? maybe negative of maxint? This would save us a an API.

Reply:
no unfortunately Vault does not support infinite TTL

Reply:
see my initial comment about a flag for distinguish static and dynamic role.